tests/server/test_cli.py

```diff
@@ -37,6 +37,10 @@ async def test_libertem_server_cli_startup():
         if 'LiberTEM listening on' in line:
             break
 
+    addr = line[line.find("http"):].strip()
+    async with http_client.get(addr) as response:
```

Comment:
I think this should work:

```suggestion
    async with http_client.get(addr, timeout=30.0) as response:
```

Reply:
If the check were moved into the `try/except/finally` block below would that do the job? The assert would send us via `except` i.e. `p.terminate() ; p.kill()`

Reply:
Yup, that seems to work! Tested by setting the timeout to a very small non-zero value, and no `libertem-server` seems to be running afterwards.